### ISSUE TITLE
open source the KICAD files?

### ISSUE THREAD

**Issue by KoSv** [erica-synths/diy-eurorack#9]
Hi there, 

where can I find the kicad files? 
This would be really helpful to have them since some pcb suppliers need modified versions of the drill files (and there are some other issues too). 

Cheers!
Konstantin

**Comment by truedat101** [comment]
Yes, I noted that the schematics in the PDFs for each of these modules references Kicad: 
![image](https://user-images.githubusercontent.com/64202/113175047-dddf0280-91ff-11eb-82ee-ebfe0c1ff355.png)

I'm wondering what the point of not releasing the Kicad file was.
